scripts/run_examples

```diff
@@ -11,22 +11,22 @@ readonly DOCKER_USER="${USER:-root}"
 readonly SCRIPTS_DIR="$(dirname "$0")"
 
 # Build Oak server.
-"$SCRIPTS_DIR/docker_run" "$SCRIPTS_DIR/build_server"
+"$SCRIPTS_DIR/build_server"
 
 # Run Rust-based Oak examples, each with their own Oak server instance.
 readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module/rust$' | cut -d'/' -f2)"
 for example in ${RUST_EXAMPLES}; do
-  container_id=$("$SCRIPTS_DIR/docker_run" --detach ./bazel-bin/oak/server/asylo/oak)
-  docker logs --follow "$container_id" &
-  "${SCRIPTS_DIR}/docker_run" "${SCRIPTS_DIR}/run_example" "${example}"
-  docker stop "$container_id"
+  ./bazel-bin/oak/server/asylo/oak &
+  SERVER_PID=$!
+  "${SCRIPTS_DIR}/run_example" "${example}"
```

Comment:
Do we need a `trap cleanup_fn EXIT` to ensure that if `run_example` fails (and thus terminates the script) the running server still gets tidied up?

Reply:
Added TODO.